pkg/platform/aws/aws.go

```diff
@@ -287,6 +288,52 @@ func (c *config) checkValidConfig() hcl.Diagnostics {
 	return diagnostics
 }
 
+// checkLBPortsUnique checks if user is using multiple worker
+// pools, then they should have lb_http_port and lb_https_port flag set
+// with different values.
```

Comment:
```suggestion
// checkLBPortsUnique checks that the lb_http_port and lb_https_port
// flags have different values if the user is using multiple worker
// pools.
```

Reply:
Well, technically, if you set `lb_http_port` and `lb_https_port` to the same port on single worker pool, it will also trigger an error :smile: